.travis.yml

```diff
@@ -10,12 +15,14 @@ env:
     - MOZDOWNLOADPARAMS='-t release -v latest'
     - MOZDOWNLOADPARAMS='-t release -v latest-esr'
 
-before_install:
-  - sudo apt-get update -qq
+addons:
+  apt:
+    packages:
+    - ant
+    - python-pip
 
 install:
-  - sudo apt-get install -y ant python-pip
-  - sudo pip install mozdownload==1.12
+  - pip install --user mozdownload==1.12
```

Comment:
oh wow. please change that to `mozdownload==1.19`! 1.12 will definitely fail to download the Firefox build.